src/components/Modals/UploadButtonGroup.jsx

```diff
@@ -1,12 +1,14 @@
 // React Imports
-import React from 'react';
+import React, { useState } from 'react';
+
 // Material UI Imports
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import PhotoCameraIcon from '@mui/icons-material/PhotoCamera';
 import SearchIcon from '@mui/icons-material/Search';
-import useMediaQuery from '@mui/material/useMediaQuery';
-import { useTheme } from '@mui/material/styles';
+import WebcamModal from './WebcamModal';
+// import useMediaQuery from '@mui/material/useMediaQuery';
+// import { useTheme } from '@mui/material/styles';
```

Comment:
While there are other functions that would be used to detect camera/webcams, these media query functions and theming would be of great help for the styling. Think we could leave them on until we come out with a workable design that encompasses both this PR and previous one.

Reply:
I agree